### PR TITLE
update broadcast.create|update endpoints

### LINKED ISSUE #64: update `broadcast.create|update` endpoints
Just finished updating the docs for them:
https://lichess.org/api#tag/Broadcasts/operation/broadcastTourCreate
https://lichess.org/api#tag/Broadcasts/operation/broadcastTourUpdate

berserk still uses the old forms.

### PR DESCRIPTION
**Pull request by AAArmstark**
Close #64

I wanted to give this a try and hope that not too much is wrong. Tested multiple times and worked so far.

### CODE REVIEW

File: berserk/clients/broadcasts.py [hunk 1]
```diff
@@ -25,23 +25,32 @@ def create(
         self,
         name: str,
         description: str,
+        auto_leaderboard: bool,
         markdown: str | None = None,
-        official: bool = False,
+        tier: int | None = None,
+        players: Any | None = None,
```
Comment (kraktus) [inline berserk/clients/broadcasts.py:31]: Players field should be a typed dict with 4 fields, source_name, display_name, rating and title (both optional). Then would need an internal function to convert them to the string

File: berserk/clients/broadcasts.py [hunk 2]
```diff
@@ -60,9 +69,10 @@ def update(
         broadcast_id: str,
         name: str,
         description: str,
+        auto_leaderboard: bool,
         markdown: str | None = None,
-        official: bool = False,
-        slug: str = "-",
+        tier: int | None = None,
+        players: Any | None = None,
```
Comment (kraktus) [inline berserk/clients/broadcasts.py:75]: Same

**Review by kraktus — COMMENTED** [review]
Thanks for the PR! I’ve left a comment for better typing, if you can do it that’s perfect but if not it’s good too, I’ll do it later.

**Comment by AAArmstark** [comment]
Thanks! I have tried to make the changes, but without further guidance or an example I won't be able to do it correctly. So it is probably better if you make the changes, as it will take much longer to explain it to me.

**Comment by kraktus** [comment]
can see how I did it here: https://github.com/lichess-org/berserk/pull/67/commits/334bf3aa9c1eb0e3ca8b517c51f35abc2bcf8be2 feel free to post questions here or elsewhere if you want :) cheers

**Pull request merged**